modules/persistent-storage-csi-group-snapshots-create.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-group-snapshots-create_{context}"]
+= Creating a volume group snapshot
+
+When you create a `VolumeGroupSnapshot` object, {product-title} creates a volume group snapshot.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* Enabled this feature using feature gates. For information about how to use feature gates, see _Enabling features sets by using feature gates_.
+* The persistent volume claims (PVCs) that you want to group for the snapshot have been created using a CSI driver that supports `VolumeGroupSnapshot` objects.
+* A storage class to provision the storage back end.
+* Administrator has created the `VolumeGroupSnapshotClass` object.
+* No pods are using the PVC that you want to take a snapshot of.
```

Comment:
I don't recall this being a limitation. Where does this come from?

Reply:
right agreed i already commented on this, i thought LIsa fixed it. I think we can reuse the same wording 
"No guarantees of application consistency, for example, crash consistency, are provided beyond those provided by the storage system"

Reply:
@gnufied @gcharot This statement regarding in-use PVCs is in the regular snapshot documentation too. Should it be removed from there as well? 

@gcharot The statement you mention "No guarantees...", do you want this put in the Prerequisites section? It's already in the Limitations section. I'm not clear why would need it here as well.

Reply:
So while it's obviously safer to have volume not in-used, it not's a prerequisite. 
I would remove this one and keep the one in the limitation section.

Reply:
Done.